test/fixtures/app.jsx

```diff
@@ -9,4 +9,5 @@ const mycomp = () => (
     <Trans i18nKey="key4" count={count}>You have {{count}} apples</Trans>
     <Trans i18nKey="key5">You have <a>one <i>very</i> bad</a> apple</Trans>
     <Trans i18nKey="key6">This is a <strong>{{test}}</strong></Trans>
+    <Trans>key7 default</Trans>
```

Comment:
I suggest adding a test for this:

```
<Trans>We can use Trans without i18nKey="..." as well!</Trans>
```

That makes sure keyRegExp will never match the element content.